crates/edr_solidity/Cargo.toml

```diff
@@ -5,4 +5,4 @@ edition = "2021"
 
 [dependencies]
 edr_eth = { version = "0.2.0-dev", path = "../edr_eth" }
-revm = { git = "https://github.com/Wodann/revm", rev = "93d8ebe", version = "3.3", default-features = false, features = ["dev", "serde", "std"] }
+revm = { version = "6.1", default-features = false, features = ["dev", "serde", "std"] }
```

Comment:
Should we use the same `revm` version in all crates? I noticed here it's the released `6.1` while earlier it was still unreleased `6.2-dev`